### PR TITLE
Fix/SKU selector classname

### PR DESCRIPTION
**Pull request by EmanuelRamos115**
#### What does this PR do? \*

Adds a class to the selected sku to make it possible to change its color for other stores with different color palettes

#### How to test it? \*

There is no way to test for sure, you would just use the class to change the color when necessary

### CODE REVIEW

File: react/AutocompleteBlock.tsx [hunk 1]
```diff
@@ -341,7 +342,7 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
                             return (
                               <span
                                 key={item.itemId}
-                                className={`mr4 ${handles.skuSelection}`}
+                                className={`mr4 ${handles.skuSelection} ${item.itemId === selectedItem.value && handles.skuSelectionSelected}`}
```
Comment (ataideverton) [inline react/AutocompleteBlock.tsx:345]: The goal here is to just add a `true` or `false` to the className?

**Review by ataideverton — CHANGES_REQUESTED** [review]
Add CHANGELOG

File: react/AutocompleteBlock.tsx (hunk 1 above)
Comment (EmanuelRamos115) [reply react/AutocompleteBlock.tsx:345]: Yes! Only to get when is selected

**Comment by EmanuelRamos115** [comment]
> Add CHANGELOG

**Review by ataideverton — APPROVED** [review]

**Pull request merged**